website/docs/r/postgresql_virtual_network_rule.html.markdown

```diff
@@ -88,7 +88,7 @@ The following arguments are supported:
 
 The following attributes are exported:
 
-* `id` - The ID of the SQL virtual network rule.
+* `id` - The ID of the PostgreSQL Virtual Network Rule.
```

Comment:
Whoops, sorry @tombuildsstuff my bad.

Reply:
all good - I missed it on the first pass through :)